src/dto/proposals-query.js

```diff
@@ -18,12 +18,16 @@
 // @flow
 import type { HttpQueryParams } from '../adapters/interface'
 
-class ProposalsFilter {
-  providerId: ?string
```

Comment:
+1 Look you found a way to format query.
Would be nice if to support all API supported fields. Instead of depreciating it.